html/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/StyleSheet.kt

```diff
@@ -38,12 +41,22 @@ class CSSRulesHolderState : CSSRulesHolder {
  * ```
  */
 open class StyleSheet(
+    prefix: String?,
```

Comment:
Does it make sense in your opinion to name this constructor parameter something like `customNamePrefix` or just `customPrefix`?
So it's not confused with `protected val prefix: String`... 

Then it will be easier with this line:
`val usePrefix: Boolean = prefix == null` // by looking at this line it's not clear right away what `prefix` is meant here (from constrcutor of the property)

Reply:
How about `selectorsPrefix`?

Reply:
renamed to `customPrefix` :)